ci-operator/step-registry/openshift/microshift/cryptoscan/openshift-microshift-cryptoscan-commands.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -euo pipefail
+
+source "${SHARED_DIR}/ci-functions.sh"
+
+cp /secrets/import-secret/.dockerconfigjson ${HOME}/.pull-secret.json
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)